doozer/doozerlib/backend/konflux_fbc.py

```diff
@@ -310,6 +311,16 @@ async def rebase(self, metadata: ImageMetadata, bundle_build: KonfluxBundleBuild
             if self._record_logger:
                 self._record_logger.add_record("rebase_fbc_konflux", **record)
 
+    async def _get_referenced_images(self, metadata: ImageMetadata, bundle_build: KonfluxBundleBuildRecord):
+        assert bundle_build.operator_nvr, "operator_nvr is empty; doozer bug?"
+        nvrs = {bundle_build.operator_nvr}
+        if bundle_build.operand_nvrs:
+            nvrs |= set(bundle_build.operand_nvrs)
+        assert metadata.runtime.konflux_db is not None, "konflux_db is not initialized. Doozer bug?"
+        konflux_db: KonfluxDb = metadata.runtime.konflux_db
+        ref_builds = await konflux_db.get_build_records_by_nvrs(list(nvrs))
```

Comment:
I think this assumes that the db is bound to the right table? maybe assert here?
or call the .bind again and revert after operation?

Reply:
Good point! I will fix this in the next push.

Reply:
I updated to use assert to ensure the right table is bound. Binding again might cause race condition without locking if it is not the right table.